SimMuon/GEMDigitizer/src/ME0ReDigiProducer.cc

```diff
@@ -128,25 +131,32 @@ void ME0ReDigiProducer::buildDigis(const ME0DigiPreRecoCollection & input_digis,
       if (reDigitizeOnlyMuons_ and fabs(me0Digi.pdgid()) != 13) continue;
       if (!reDigitizeNeutronBkg_ and !me0Digi.prompt()) continue;
 
-      // scale for luminosity
-      if (CLHEP::RandFlat::shoot(engine) > instLumi_*1.0/10) continue;
-
+      // scale background hits for luminosity
+      if (!me0Digi.prompt())
+	if (CLHEP::RandFlat::shoot(engine) > instLumi_*1.0/5) continue;
```

Comment:
Why are you dividing by 5 here a not by 10. I believe this line should not have changed. Instead in the configuration the instLumi parameter should have changed from 5 to 10.